packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/assetExplorer/assetTable/assetTableColumnDefinitionsFactory.tsx

```diff
@@ -87,7 +87,14 @@ export class AssetTableColumnDefinitionsFactory {
       id: 'creationDate',
       header: 'Creation Date',
       cell: ({ creationDate }) =>
-        creationDate ? getFormattedDateTime(creationDate) : '-',
+        creationDate ? (
+          <DateTime
+            dateTime={creationDate.getTime()}
```

Comment:
Is it possible to change creationDate to a number prop? I don't think we should do a bunch of conversions number -> date -> number ->date. I think all of the places we have dates (except for probably viewport) should start as numbers.